data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
```

Comment:
This comment is just a description of standard style; secondly it's not a comment, it's a string. Just so you know, Python only understands these to be documentation when you put them in certain positions (like the start of a module, class, or function). Also there should only one blank line between previous.